### ISSUE TITLE
Verified Status an additional option

### ISSUE THREAD

**Issue by duiveldoder** [mapbox/osmcha-frontend#679]
Right now we can for the "verify" on the top right only select "good" or "bad"

As I keep track on changes in my city - all changes - of course I wish to not go over anything that has been verified already again, so I wish to be able to filter out the "verified" which OSMCha already allows.

But - this kinda mandates me to flag it either as "good" or "bad"

But what if it's actually a "good" commit/changeset, but I still have some questions left in form of comments on the changeset?
Like - if the mapper won't respond within 2 or 3 months - I'll revert that specific issue - but let everything else I deem correct in place since 99% is a good edit.. it's just one issue I have for which I made a comment - and just waiting for the feedback of the mapper.

I also wish of course in OSMCha to simply look at those then - where is there still something left to do?

But that's not the same as "unverified" of course.. I did verify the changeset already - just waiting on feedback.

So how about having more options than good or bad?

Like for instance:"Feedback" then we know on that changeset we're waiting for feedback - and if the mapper doesn't respond in 2 or 3 months - for instance if my question is:"what is your source, is it a legal source?"

Then if the mapper refuses to answer - we can easily filter such out by having the "feedback" or "on hold" or "waiting" or whatever option we could use to point out it's neither good nor bad, we just need to make sure on something and decide later ..

yeah any questions I'd be more than happy to answer..

note this request is directly related also to https://github.com/mapbox/osmcha-frontend/issues/306

which also points out the "good" and "bad" options we have now - isn't cutting it in all situations.. We need some additional choices there in the "verified" option

**Comment by rusty-snake** [comment]
Going into a similar direction. IDK if you are aware of the different choice Gerrit offers for code review (https://gerrit-review.googlesource.com/Documentation/config-labels.html#label_Code-Review) but I would like to have a similar concept in OSMCha:

- `-2`: Bad changeset. I am sure that at least one change in this changeset is bad. This could be a bad tagging (e.g. unconnected highway, typos in tag/value, ...) or a correct tagging of a non-exiting reality (i.e. OTG looks different, use source was outdated/wrong).
- `-1`: Suspicious changeset. I am not sure that the changeset is bad, but at least one change looks suspicious (e.g. name-tag abuse, ...) but without survey/local knowledge I can not tell fore sure.
- `+1`: Looks good to be but I can not fully verify. Either because the (some) touched objects are from a tagging domain I rarely work (e.g. I'm a bicyle mapper but the changeset changed railway infrastructure) or because you need survey/local knowledge (e.g `smoothness`).
- `+2`: Good. I know the tagging and the area.
